### DISCUSSION TITLE
Introducing clone-wp-theme: A New CLI Tool for Flynt Theme Setup

### DISCUSSION THREAD

**Discussion by rajpatel2435** [flyntwp/flynt#573] [Show and tell]
Hi everyone,

@timohubois @aaronmeder

I’m excited to share clone-wp-theme, a new CLI tool designed to streamline WordPress theme setup. With this tool, you can easily choose and clone predefined themes like Flynt and Sage-Vite.

Features:
Interactive Setup: Choose from Flynt or Sage-Vite and automatically clone the corresponding repo.

Simple Installation: Install globally with npm install -g clone-wp-theme and start with clone-wp-theme

Check out the [README](https://github.com/rajpatel2435/create-wp-theme) for more details and instructions.

https://www.npmjs.com/package/clone-wp-theme

I’d love your feedback and contributions. Let’s make theme development easier together!

Best,
Raj Patel

**Comment by rajpatel2435** [comment]
Hi @steffenbew @timohubois @roelti 

Thank you for your support! I’m thrilled you find clone-wp-theme helpful. Integrating it directly into the main Flynt theme would be an exciting opportunity and could streamline theme setup even more for Flynt users.

I’d be happy to explore contributing this tool to the core Flynt theme itself. It would allow developers to kickstart projects even faster, all within the Flynt framework. If the Flynt team is open to it, I can create a pull request, and we can work together to make it fit seamlessly with the existing workflow.

Let me know your thoughts on this, and I’m excited about the possibility of contributing directly to the Flynt theme!

Best regards,
Raj Patel

**Reply by rajpatel2435** [reply]
Hi @steffenbew @timohubois @aaronmeder 

Kindly provide an update!! if you want I can also make separate npm package for Flynt theme?

Thanks!

**Comment by rajpatel2435** [comment]
Hi @steffenbew @timohubois @aaronmeder,

Just following up on my previous message regarding integrating the clone-wp-theme CLI tool with the Flynt theme. I’d love to hear your thoughts on how we could move forward with this collaboration or if there's anything you'd like me to adjust or elaborate on.

Thank you, and I look forward to your response!

Best regards,
Raj Patel

**Comment by timohubois** [comment]
@rajpatel2435 we really appreciate your enthusiasm and interest in integrating the clone-wp-theme CLI tool with the Flynt theme. However, we aim to keep the core of Flynt as clean and lightweight as possible to maintain flexibility and simplicity for all users.

That said, we think your tool could be a fantastic addition to the Flynt ecosystem! A great place for it might be the [Flynt Add-Ons repository](https://github.com/awesome-flynt/flynt-add-ons), where it can stay independent from the core.

Let us know what you think?

Best,
Timo

**Comment by rajpatel2435** [comment]
Hi @timohubois,

This CLI tool doesn't add any extra packages; it simply clones your GitHub repository. I don’t believe it will affect the performance, cleanliness, or lightness of your code in any way.

Best,
Raj

**Reply by timohubois** [reply]
@rajpatel2435 we aim to keep the core as clean and lean as possible. Adding an additional tool to the core introduces a dependency, which would require proper explanation, documentation, and maintenance. It also adds an extra layer of complexity. For these reasons, I believe the suggested repository is the most suitable place for this tool.
